app/views/directives/notifications/notification-body.html

```diff
@@ -5,6 +5,8 @@
   <a
     class="pull-right"
     href=""
+    ng-if="!notification.actions.length"
+    tabindex="0"
```

Comment:
Remove `tabindex`. It's not needed now that we have an `href`. Looks like this was accidentally added back on rebase

Reply:
Ok, removed `tabindex`. -thanks